chart/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: ssl-exporter
+description: A Helm chart for ribbybibby/ssl_exporter
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "2.4.2"
```

Comment:
This is version of what? I do not understand.

Reply:
It's the version of [ribbybibby/ssl_exporter](https://github.com/ribbybibby/ssl_exporter)

Reply:
Ok, but i am not sure if we should have this version here.

Reply:
As I understand it, appVersion should be the version of the app we deploy, in this case it's the version of ssl-exporter.